docs/user-guide/review.md

```diff
@@ -0,0 +1,48 @@
+# Crawl Review
+
+The Crawl Review page provides a streamlined interface for assessing the capture quality of pages within an archived item using the heuristics collected during crawl analysis.
+
+Crawls can only be reviewed once [crawl analysis](archived-items.md#crawl-analysis) has been run. If multiple analysis runs have been completed, the page analysis heuristics will be used from the selected analysis run, which are displayed next to the archived item name. The most recent analysis run is selected by default, but you can choose to display data from any other completed or stopped analysis run here as well.
+
+## Heuristics
+
+Crawl analysis generates comparisons across three heuristics that can indicate which pages may be the most problematic.
+
+### Screenshot Comparison
+
+Screenshots taken during crawl analysis are compared with the page thumbnail created during crawling. Screenshots are compared using [Pixelmatch](https://github.com/mapbox/pixelmatch).
```

Comment:
Why note that we use Pixelmatch here?

Reply:
I think it's useful for folks to know how these scores are generated should they wish to test or develop similar thngs themselves.  Elsewhere in the docs we note the method behind text comparison, and that we use zxcvbn to derive password strength.  Personally I like these little details!  They provide extra context as to how the software was developped in a way that is more accessible for non-programmers

Reply:
Gotcha, yeah I can see the value in that. If this is aimed towards non-developers we should summarize the mechanism directly instead of linking to a github page. Especially for pixelmatch, the README is very technical.

How about something like:
```suggestion
Screenshots are compared by measuring the perceived difference between color samples and by the intensity of difference between pixels. These metrics are provided by the open-source tool [Pixelmatch](https://observablehq.com/@mourner/pixelmatch-demo).
```